cmd/xl-v1-multipart.go

```diff
@@ -392,20 +400,21 @@ func (xl xlObjects) PutObjectPart(ctx context.Context, bucket, object, uploadID
 	defer postUploadIDLock.Unlock()
 
 	// Validate again if upload ID still exists.
-	if !xl.isUploadIDExists(bucket, object, uploadID) {
-		return pi, errors.Trace(InvalidUploadID{UploadID: uploadID})
+	if !xl.isUploadIDExists(ctx, bucket, object, uploadID) {
+		logger.LogIf(ctx, InvalidUploadID{UploadID: uploadID})
```

Comment:
No need to log here since `isUploadIDExists` already takes care of the `InvalidUploadID` error condition.

Reply:
done